app/views/idv/doc_auth/_back.html.erb

```diff
@@ -0,0 +1,27 @@
+<%#
+locals:
+* action: (Optional) Flow action to call to return to the previous step.
+* step: (Optional) Name of step to which user should be returned.
+* path: (Optional) Path to which user should navigate.
+%>
+<%
+if [:action, :step, :path].all? { |key| !local_assigns[key] }
+  raise 'must pass one of action, step, or path'
+end
```

Comment:
The comment makes it look like all 3 are optional, but the check makes it clear one is required, maybe just comment "must provide one of..." up top?

Reply:
> The comment makes it look like all 3 are optional, but the check makes it clear one is required, maybe just comment "must provide one of..." up top?

Updated in 45ea894.